sdks/python/apache_beam/runners/worker/channel_factory.py

```diff
@@ -23,8 +23,24 @@
 
 class GRPCChannelFactory(grpc.StreamStreamClientInterceptor):
   DEFAULT_OPTIONS = [
-      ("grpc.keepalive_time_ms", 20000),
-      ("grpc.keepalive_timeout_ms", 300000),
+      # Default: 30000ms (30s), increased to 180s to reduce ping frequency
```

Comment:
Are the pings expensive? Does this mean that it will take 180s instead of 30s to notice that the runner terminated?  This doesn't seem good for streaming pipelines if the case.